### ISSUE TITLE
Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.

### ISSUE THREAD

**Issue by lucasjinreal** [microsoft/DeepSpeed#4021]
Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.

**Comment by FrankWhh** [comment]
I just also meet. when I finetune LLM with 1 or 2 gpus it's ok， but meet this issue when gpus > 3. I don't know why!

**Comment by lucasjinreal** [comment]
@FrankWhh  what the model are u finetruning? is it llama2?

**Comment by FrankWhh** [comment]
no, i finetune glm2 and baichuan with lora.

**Comment by lucasjinreal** [comment]
I haven't get this error with baichuan model though, can u share your lr configuration and deepspeed config?

**Comment by FrankWhh** [comment]
deepspeed config:
{
  "train_micro_batch_size_per_gpu": "auto",
  "zero_allow_untested_optimizer": true,
  "fp16": {
    "enabled": "auto",
    "loss_scale": 0,
    "initial_scale_power": 16,
    "loss_scale_window": 1000,
    "hysteresis": 2,
    "min_loss_scale": 1
  },
  "zero_optimization": {
    "stage": 2,
    "allgather_partitions": true,
    "allgather_bucket_size": 5e8,
    "overlap_comm": false,
    "reduce_scatter": true,
    "reduce_bucket_size": 5e8,
    "contiguous_gradients" : true
  },

"flops_profiler_config":true,
"gradient_accumulation_steps": "auto",
"gradient_clipping": "auto"
}

lr : 5e-5  
lr_scheluder : cosine
torch 2.0 + transformers 4.29.1 + deepspeed 0.10.0 
V100
------
1 or 2 gpu ok, while >=3 fail

**Comment by lucasjinreal** [comment]
In my experience, if you using 3 v100 fail, then 1 or 2 will also be fail sooner or later. 
v100 can only using fp16 training not bf16 which is cubersome on overflow issue

**Comment by FrankWhh** [comment]
NO, 1 or 2 gpu is ok, it will fininsh the training. That's where i'm so confused

**Comment by lucasjinreal** [comment]
Then that was out of expectation, in theory, more GPUs get more large batchsize, training should be more stable.

**Comment by tjruwase** [comment]
@lucasjinreal, can you please share more details to help understand and repro this issue? Thanks!
